src/components/staking/LiValidator.vue

```diff
@@ -49,7 +49,7 @@
       </div>
     </td>
     <td :class="{ 'hide-xs': showOnMobile !== 'expectedReturns' }">
-      {{ validator.expectedReturns | bigFigureOrShortDecimals }}
+      {{ validator.expectedReturns | bigFigureOrShortDecimals }} %
```

Comment:
expected returns are in percent value. so they are delivered as `0.01` = `1%`. this would display only `0.01 %`

Reply:
you need a bigFigureOrPercent function

Reply:
thanks